Description: run nvidia-prime setup on login
Author: Jonathan Riddell <jriddell@ubuntu.com>
Origin: me
Bug: https://bugs.launchpad.net/ubuntu/+source/nvidia-prime/+bug/1428328
Bug-SDDM: https://github.com/sddm/sddm/issues/393
Forwarded: yes, although it's not very elegant so may not be accepted
Last-Update: 2015-04-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/data/scripts/Xsetup
+++ b/data/scripts/Xsetup
@@ -1,3 +1,7 @@
 #!/bin/sh
 # Xsetup - run as root before the login dialog appears
 
+if [ -e /sbin/prime-offload ]; then
+    echo running NVIDIA Prime setup /sbin/prime-offload
+    /sbin/prime-offload
+fi
--- a/data/scripts/Xstop
+++ b/data/scripts/Xstop
@@ -1,2 +1,7 @@
 #!/bin/sh
 # Xstop - run as root after stopping X
+
+if [ -e /sbin/prime-switch ]; then
+    echo running NVIDIA Prime switch /sbin/prime-switch
+    /sbin/prime-switch
+fi
